march_hardware/src/MarchRobot.cpp

```diff
@@ -60,6 +70,15 @@ void MarchRobot::stopEtherCAT()
   ethercatMaster.stop();
 }
 
+void MarchRobot::resetIMC()
+{
+  ROS_INFO("Resetting all IMC on initialization");
```

Comment:
```suggestion
  ROS_INFO("Resetting all IMCs");
```
Its not really relevant whether it is during initialization or not